src/SignInComponent.tsx

```diff
@@ -0,0 +1,342 @@
+import React, { useState } from 'react';
+import { css } from "@emotion/core";
+import { body, headline } from '@guardian/src-foundations/typography/cjs';
+import { palette } from "@guardian/src-foundations";
+import { Button } from "@guardian/src-button";
+import { from, until } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { SvgCross } from '@guardian/src-icons';
+import { SvgInfo } from '@guardian/src-icons';
+import { AppStore } from "./assets/app-store"
+import { PlayStore } from "./assets/play-store"
+
+const imgHeight = "300";
+
+export type Props = {
+  message: string;
+  onButtonClick: (buttonType: string) => void;
+  firstName?: string;
+};
+
+export const wrapper = css`
+  html {
+    box-sizing: border-box;
+  }
+  *,
+  *:before,
+  *:after {
+    box-sizing: inherit;
+  }
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: ${palette.background.secondary};
+  color: ${"#333"};
```

Comment:
Is there a palette colour value we could use here?

Reply:
That colour came from the mockup provided by UX, but I couldn't find it in any palettes from Source - would you generally go with the colour provided by UX or find something similar from the palettes?

Reply:
I think my first thought would be to see if there's something similar we can use in the source palette. It's definitely okay to use custom colours (though I'd vote to pull them out and give them a name) but nice if we can use source colours for consistency!

Reply:
I managed to dig one out of Source in the end anyway - included now.